alchemist-maps/src/main/java/it/unibo/alchemist/boundary/gpsload/impl/TraceLoader.java

```diff
@@ -58,6 +59,7 @@
         .withNarrowingConversions()
         .withAutoBoxing()
         .build();
+    private static final Semaphore SEMAPHORE = new Semaphore(1);
```

Comment:
A semaphore with a single permit is better called `MUTEX`

Reply:
done